core/src/main/java/com/datastrato/gravitino/storage/relational/utils/SessionUtils.java

```diff
@@ -63,6 +63,25 @@ public static <T, R> R doWithCommitAndFetchResult(Class<T> mapperClazz, Function
     }
   }
 
+  /**
+   * This method is used to perform a database operation without a commit and fetch the result. If
+   * the operation fails, will throw the RuntimeException.
+   *
+   * @param mapperClazz mapper class to be used for the operation
+   * @param func the operation to be performed with the mapper
+   * @return the result of the operation
+   * @param <T> the type of the mapper
+   * @param <R> the type of the result
+   */
+  public static <T, R> R doWithoutCommitAndFetchResult(Class<T> mapperClazz, Function<T, R> func) {
+    try {
+      T mapper = SqlSessions.getMapper(mapperClazz);
+      return func.apply(mapper);
+    } catch (Throwable t) {
+      throw t;
```

Comment:
Do you really need this? what's the purpose?

Reply:
just keep align with other places. 

https://github.com/datastrato/gravitino/blob/2bf1767ab71da9b1f55253de92c055899a82e515/core/src/main/java/com/datastrato/gravitino/storage/relational/utils/SessionUtils.java#L79

Reply:
I mean: Since you catch a throwable t and then do nothing, and throw it again, do we still need the catch block?

Reply:
i will polish it